kinto/integration-test/test_kinto_sync.py

```diff
@@ -0,0 +1,156 @@
+import base64
+import datetime
+import re
+import requests
+import time
+import unittest
+
+from kinto_client import Client
+from kinto_client import replication
+
+class Kinto_Sync(unittest.TestCase):
+    """
+    Tests that given two Kinto servers you can successfully sync data
+    from one to the other
+    """
+
+    def setUp(self):
+        self.master_url = "http://192.168.99.100:8888/v1/"
+        self.read_only_url = "http://192.168.99.100:8889/v1/"
```

Comment:
Not sure if we can get these URLs via docker or not, but may be better than hard coding them.